dso-l1/src/main/java/com/tc/object/DistributedObjectClient.java

```diff
@@ -445,9 +455,11 @@ private void waitForHandshake() {
     this.clientHandshakeManager.waitForHandshake();
     if (this.channel != null) {
       final TCSocketAddress remoteAddress = this.channel.getRemoteAddress();
-      final String infoMsg = "Connection successfully established to server at " + remoteAddress;
-      if (!this.channel.getProductId().isInternal()) {
-        DSO_LOGGER.info(infoMsg);
+      if(remoteAddress!=null){
```

Comment:
If remote address is null, there is probably a larger problem at hand.  What is driving this change?